services/provider/server/server.go

```diff
@@ -189,3 +205,150 @@ func newClient() (client.Client, error) {
 
 	return client, nil
 }
+func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerResource *ocsv1alpha1.StorageConsumer) ([]*pb.ExternalResource, error) {
+	var extR []*pb.ExternalResource
+
+	// Configmap with mon endpoints
+	configmap := &v1.ConfigMap{}
+	err := s.client.Get(ctx, types.NamespacedName{Name: monConfigMap, Namespace: s.namespace}, configmap)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get %s configMap. %v", monConfigMap, err)
+	}
+
+	// Get address of first mon from the monConfigMap configmap
+	cmData := strings.Split(configmap.Data["data"], ",")
+	if len(cmData) == 0 {
+		return nil, fmt.Errorf("configmap %s data is empty", monConfigMap)
+	}
+
+	extR = append(extR, &pb.ExternalResource{Name: monConfigMap, Kind: "ConfigMap", Data: mustMarshal(map[string]string{
```

Comment:
Can we split the params across lines to make this line more readable?

```suggestion
extR = append(extR, &pb.ExternalResource{
	Name: monConfigMap, 
	Kind: "ConfigMap", 
	Data: mustMarshal(map[string]string{
		"data":     cmData[0], // Address of first mon
		"maxMonId": "0",
		"mapping":  "{}",
	})
})
```